test/framework/resources/templates/logger-app.yaml.tpl

```diff
@@ -18,7 +18,7 @@ spec:
     spec:
       containers:
       - name: logger
-        image: k8s.gcr.io/logs-generator:v0.1.1
+        image: eu.gcr.io/gardener-project/3rd/k8s_gcr_io/logs-generator:v0.1.1
```

Comment:
Hmm, `k8s.gcr.io/logs-generator:v0.1.1` is already on GCR. Do we need to copy it?

Reply:
Good question. I only did it because there copies of other images of k8s.gcr.io in `eu.gcr.io/gardener-project/3rd/k8s_gcr_io`. What's the reason for these copies then?

Reply:
> What's the reason for these copies then?

I don't know. But I guess, it doesn't matter anyways...